music21/configure.py

```diff
@@ -1479,8 +1490,8 @@ def getDialogs(self):
 
     def _introduction(self):
         msg = []
-        msg.append('Welcome to the music21 Configuration Assistant. You will be guided '
-                   + 'through a number of questions to install and set up music21. '
+        msg.append('Welcome the music21 Configuration Assistant. You will be guided '
+                   + 'through a number of questions to install and setup music21. '
```

Comment:
I'm curious why this unrelated change? Also "set up" is the correct verb phrase here; "setup" is a noun.

Reply:
I did not (intentionally) make any such change.